### DISCUSSION TITLE
Mixings between versions names v10.1.1 and v10.2.0

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#18673] [Feature Requests]
### Summary

As of today:

- Installation latest from scratch installs v10.1.1 
- Updating a 10.0.0  installs v10.2.0

### Motivation

-

### Detailed Design

-

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

**Comment by Pachat** [comment]
Any way. v10.2.0 was unusable because event the current user could not modify its profile, event with all permissions in all collections including Directus ones.
I had to revert to my last version: v10.0.0

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/issues/18632
